.github/workflows/bug_snapshot.yaml

```diff
@@ -9,6 +9,9 @@ name: Bug Snapshot
 on:
   workflow_dispatch:
     branches: [ main ]
+  schedule:
+    # Run daily at 14:05
+    - cron: '0 14 * * 8'
```

Comment:
Is this right? Looks like 14:00... not sure what 8 means there, should be day of week 0 to 6 I think